### DISCUSSION TITLE
Show count of a eloquent in table

### DISCUSSION THREAD

**Discussion by mionoeldavis** [rappasoft/laravel-livewire-tables#993] [Q&A]
Hi, i am very new to livewire tables and cant find a solution for my problem.

I have a table with users, now i want to have a column where it shows the post count of each user. How can i do it?

**Comment by lrljoe** [comment]
Add the following to your Builder instance, or against your model (however you prefer)

`withCount('posts')`

Then reference it in your table column definition in one of the following ways:


```
Column::make('Number of Posts', 'posts_count')->label(fn ($row) => $row->posts_count),

```

Or you can do it as follows if you want it to be sortable:

```
  Column::make('Number of Posts', 'id')
      ->format(
          fn ($value, $row, Column $column) => intval($row->posts_count)
      )
      ->sortable(fn (Builder $query, string $direction) => $query->orderBy('posts_count', $direction)),

```
